clearblade/Users.py

```diff
@@ -77,3 +77,22 @@ def __init__(self, system, email, password):
             "email": email,
             "password": password
         }
+
+class ServiceUser(AnonUser):
+    def __init__(self, system, email, token):
+        super(ServiceUser, self).__init__(system)
+        self.credentials = {
```

Comment:
I believe we can replace this with something like `self.headers["ClearBlade-UserToken"] = token` and then we don't need the serviceAuthCheck method anymore because we can just use the parent class's checkAuth method.

Reply:
But then this part is there in authenticate, which should also be there in this function, but calling authenticate for this function doesn't make sense.

```
if self not in self.system.users:
            self.system.users.append(self)

```

Reply:
I believe that part is only there to auto log-out users when the script finishes. We don't need that for service accounts.